Added spinner to file upload in knowledge section

nikhil-krajput opened the pull request:
- Preview of the spinner with the message
![Screenshot 2024-10-16 at 11-58-03 InstructLab UI](https://github.com/user-attachments/assets/a55a0c7b-7c38-49b3-b798-fe8c961702bc)

vishnoianil commented:
@nikhil-krajput can you please signoff the commit to make the DCO job happy. If you click the "Details" link, it should give you the instructions on how to sign-off your commits.

vishnoianil commented:
@nikhil-krajput Lint job is failing as well. Locally run the `npm run lint:fix`, and it should fix your linting error.

nikhil-krajput commented:
> @nikhil-krajput Lint job is failing as well. Locally run the `npm run lint:fix`, and it should fix your linting error.

@vishnoianil  I have added the signed commits and fixed the lint error as well

vishnoianil commented on src/components/Contribute/Knowledge/UploadFile.tsx:
```diff
@@ -126,6 +131,14 @@ export const UploadFile: React.FunctionComponent<{ onFilesChange: (files: File[]
+          {isUploading && (
+            <>
+              <Spinner size="lg" />
+              <p>Uploading file</p>
```
@nikhil-krajput can you please update this text to "Uploading files to `taxonomy-knowledge-docs` repo in your github account."

nikhil-krajput replied:
@vishnoianil 
Updated text:
![Screenshot 2024-10-18 at 14-14-06 InstructLab UI](https://github.com/user-attachments/assets/2153f6c4-6fdc-486c-9cb8-52bf72d95fa8)

vishnoianil reviewed (commented):
Thanks for the PR @nikhil-krajput. PR looks good to me. I will give it a spin sometime tomorrow. Meanwhile can you please take care of the minor comment i added.

vishnoianil commented:
@nikhil-krajput PR looks good to me, but the commit history looks weird on your PR. Can you sync your git fork and pull the new changes in your local fork on your machine and rebase this PR, and that should fix your commit history.

vishnoianil commented:
@nikhil-krajput looks like the commit history is still not fixes. Ping me one slack, happy to help with this.

vishnoianil commented:
Raised PR #301 after fixing git commit history.
